### DISCUSSION TITLE
cant make anything

### DISCUSSION THREAD

**Discussion by NexusImmortal** [encode/starlette#2097] [Potential Issue] [answered]
i've been trying to download and get Stable Diffusion operational for a while and when i though i might just have got it working im able to input text but nothing more and i receive this from the prompt.

raise RuntimeError(f"File at path {self.path} does not exist.")
RuntimeError: File at path C:\Windows\System32\javascript\ui.js does not exist.

it also says

No module 'xformers'. Proceeding without it.

**Comment by NexusImmortal** [comment]
lol solved it myself i feel slow.... just had to update a pip i can't remember the command but you open up git try to update xformers with pip install xformers then it will tell you that you need to enter another command first to update the pip... im able to generate images now but i'm still not able to install xformers it give me this error

ERROR: THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.
    torch==2.0.0 from https://files.pythonhosted.org/packages/87/e2/62dbdfc85d3b8f771bc4b1a979ee6a157dbaa8928981dabbf45afc6d13dc/torch-2.0.0-cp310-cp310-win_amd64.whl (from xformers):
        Expected sha256 527f4ae68df7b8301ee6b1158ca56350282ea633686537b30dbb5d7b4a52622a
             Got        2620c02c050bd41c39fe592446cf0fb6060445706530f8415d33295740b15731

**Reply by adriangb** [reply, marked as answer]
This has nothing to do with Starlette

**Reply by euri10** [reply]
kamoulox
